iaso/api/tasks/create/instance_bulk_gps_push.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import viewsets, permissions, status
+from rest_framework.response import Response
+
+from iaso.api.instances import HasInstanceBulkPermission
+from iaso.api.org_units import HasCreateOrgUnitPermission
+from iaso.api.tasks import TaskSerializer
+from iaso.tasks.instance_bulk_gps_push import instance_bulk_gps_push
+
+
+class InstanceBulkGpsPushViewSet(viewsets.ViewSet):
+    """Bulk push gps location from Instances to their related OrgUnit.
+
+    This task will override existing location on OrgUnits and might set `None` if the Instance doesn't have any location.
+    """
+
+    permission_classes = [permissions.IsAuthenticated, HasInstanceBulkPermission, HasCreateOrgUnitPermission]
+
+    def create(self, request):
+        raw_select_all = request.data.get("select_all", True)
```

Comment:
I must insist that a serializer would make the API contract clearer and easier to understand.